ui/apps/platform/src/Containers/PolicyCategories/PolicyCategoriesList.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { SimpleList, SimpleListItem } from '@patternfly/react-core';
+
+type PolicyCategoriesListProps = {
+    // policyCategories: {
+    //     id: string;
+    //     name: string;
+    //     isDefault: boolean;
+    // }[];
```

Comment:
This reminds me, let’s collaborate on writing new endpoints and especially frontend types for backend protos when they become available, as a way to improve the information about APIs in ui/apps/platform/README.md